homatransport/src/application/WorkloadSynthesizer.ned

```diff
@@ -24,7 +24,7 @@ simple WorkloadSynthesizer like ITrafficGeneratorApp
 {
```

Comment:
Make sure the comments at the top of module are still correct. Is the workload synthesizer connected to the transport?